apps/system/lockscreen/js/widgets/clock/lockscreen_clock_widget_setup.js

```diff
@@ -44,6 +44,9 @@
         if (!result) {
           throw new Error(`Can't query element ${key} with query: ${query}`);
         }
+        if (!result.firstChild) {
```

Comment:
Since to add a dummy node (' ') is not usual, to comment it here could explain the reason better.